### PR TITLE
feature/consolerestoredevonfwide

### PR DESCRIPTION
**Pull request by denise-khuu**

### CODE REVIEW

File: runners/console/index.ts [hunk 1]
```diff
@@ -158,6 +166,21 @@ export class Console extends Runner {
         }
     }
 
+    async assertRestoreDevonfwIde(step: Step, command: Command, result: RunResult) {
```
Comment (GuentherJulian) [inline runners/console/index.ts:169]: Can't we simply use the assert method of installDevonfwIde method here?

File: documentation/Functions.md [hunk 2]
```diff
@@ -18,6 +18,17 @@ installDevonfwIde(["java","mvn"], "2020.08.001")
...
 ***
 
+### restoreDevonfwIde
+#### parameter
+1. The tools you want to install within the devonfw ide: string array
+2. Optional: The version of the ide to install
+#### example
+restoreDevonfwIde(["java","mvn"], "2020.08.001")
+#### details 
+The installation of the devonfw IDE is executed in a startup script.
```
Comment (GuentherJulian) [inline documentation/Functions.md:28]: We should also mention that this is only done in the case of katacoda

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: runners/console/index.ts [hunk 3]
```diff
@@ -52,6 +52,14 @@ export class Console extends Runner {
         return result;
     }
 
+
+    runRestoreDevonfwIde(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;    
+        result = this.runInstallDevonfwIde(step, command);
```
Comment (denise-khuu) [inline runners/console/index.ts:59]: Are those lines necessary? Or is 
let result = this.runInstallDevonfwIde(step, command) 
also possible?

Comment (GuentherJulian) [reply runners/console/index.ts:59]: These are not necessary. We can simply write return this.runInstallDevonfwIde(step, command)

**Review by GuentherJulian — APPROVED** [review]

**Pull request merged**